source/codegen/metadata_validation.py

```diff
@@ -113,6 +113,14 @@ class RULES:
         Optional("linked_params_are_optional"): bool,
         Optional("mapped-enum"): str,
         Optional("is_optional"): bool,
+        Optional("optional"): bool,
+        Optional("ctypes_data_type"): Or(str, None),
+        Optional("has_explicit_buffer_size"): bool,
+        Optional("is_list"): bool,
+        Optional("python_data_type"): str,
+        Optional("python_type_annotation"): str,
+        Optional("python_description"): str,
+        Optional("default"): Or(str, bool, None),
```

Comment:
This has been renamed to `python_default` because the enum values are Python-specific. 

However, I just noticed that this schema supports `default_value`: `Optional("default_value"): Or(str, bool, int, float, None),`

Based on that, I think we should rename this to `python_default_value` and add `int` and `float` to the list of types. That way, it will be a Python-specific override for `default_value`.

Reply:
Got it. I have created a PR in Scrapigen which renames `python_default` to `python_default_value`. 
https://github.com/ni/grpc-device-scrapigen/pull/155